api/src/nv_ingest_api/internal/extract/image/table_extractor.py

```diff
@@ -60,39 +61,51 @@ def _filter_valid_images(base64_images: List[str]) -> Tuple[List[str], List[np.n
 def _run_inference(
     enable_yolox: bool,
     yolox_client: Any,
-    paddle_client: Any,
+    ocr_client: Any,
+    ocr_model_name: str,
     valid_arrays: List[np.ndarray],
     valid_images: List[str],
     trace_info: Optional[Dict] = None,
 ) -> Tuple[List[Any], List[Any]]:
     """
     Run inference concurrently for YOLOX (if enabled) and Paddle.
 
-    Returns a tuple of (yolox_results, paddle_results).
+    Returns a tuple of (yolox_results, ocr_results).
     """
-    data_paddle = {"base64_images": valid_images}
+    data_ocr = {"base64_images": valid_images}
     if enable_yolox:
         data_yolox = {"images": valid_arrays}
+        future_yolox_kwargs = dict(
+            data=data_yolox,
+            model_name="yolox",
+            stage_name="table_extraction",
+            max_batch_size=8,
```

Comment:
Note to self: Update this max to a centralized env based location in https://github.com/NVIDIA/nv-ingest/pull/889/